spec/models/note_spec.rb

```diff
@@ -10,4 +12,15 @@
   it 'encrypts the body' do
     expect(note.body_before_type_cast).not_to eq('This is the body')
   end
+
+  context 'when the body has content' do
+    specify { expect(note).to be_filled }
+  end
+
+  context 'when the body is blank' do
+    specify 'filled is false' do
```

Comment:
Same as the context above we could let each expectation be self-documenting. It is also `#filled?` that is false but I'd rather the reader not be concerned with the predicate: `it "is filled"`

Reply:
[5eec6d4](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/645/commits/5eec6d438b6c80fc3528148e82b9b715fa7cde4d)